Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'bootsnap', require: false
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
```

Comment:
**[Optional]**

Kudos on implementing the CanCan on the posts controller and it is working properly on Post Controller methods :+1: but it would be nice if you use to implement it properly on the comments destroy method too because if I create a post then I make comment on it (which means I comment on my own post) then I won't be able to destroy my own comment but if I comment on someone else's post then I can easily remove that comment :+1: Kindly implement CanCan on your comments controller properly :blush: